examples/unwrap_script.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+""" This script will:
+- download 2-echo phase fieldmap data
+- unwrap phase difference
+- do the subtraction
+- save wrapped and unwrapped plot of first X,Y volume as myplot.png in current directory
+"""
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+import glob
+import subprocess
+
+import nibabel as nib
+
+from shimmingtoolbox.unwrap import prelude
+
+
+def main():
+
+    # Download example data
+    subprocess.run(['st_download_data testing_data'], shell=True, check=True)
+
+    # Open phase data
+    path_data = glob.glob(os.path.join('.', 'testing_data*'))[0]
+    fname_phases = glob.glob(os.path.join(path_data, 'sub-fieldmap', 'fmap', '*phase*.nii.gz'))
+    if len(fname_phases) > 2:
+        raise IndexError('Phase data parsing is wrongly parsed')
+
+    nii_phase_e1 = nib.load(fname_phases[0])
+    nii_phase_e2 = nib.load(fname_phases[1])
+
+    # Scale to phase to radians
+    phase_e1 = nii_phase_e1.get_fdata() / 4096 * 2 * np.pi - np.pi
+    phase_e2 = nii_phase_e2.get_fdata() / 4096 * 2 * np.pi - np.pi
+
+    # Open mag data
+    fname_mags = glob.glob(os.path.join(path_data, 'sub-fieldmap', 'fmap', '*magnitude*.nii.gz'))
+
+    if len(fname_mags) > 2:
+        raise IndexError('Mag data parsing is wrongly parsed')
+
+    nii_mag_e1 = nib.load(fname_mags[0])
```

Comment:
i would remove this test (we are in an example script-- so things are assumed the way they are: file names, number of files, etc.)